Makefile

```diff
@@ -89,47 +87,77 @@ ifneq ($(CHECKSTYLE),0)
 	PERL5LIB=lib/perlcritic:$$PERL5LIB perlcritic lib
 endif
 
-.PHONY: test
-ifeq ($(TRAVIS),true)
-test: run-tests-within-container
-else
-test: checkstyle test-with-database
-endif
+.PHONY: phony_explicit
+phony_explicit: 
```

Comment:
can you explain the purpose of this?

Reply:
This is the way I found to tell Makefile that implicit target is phony (I got impression that you asked for this here https://github.com/os-autoinst/openQA/pull/2376#discussion_r331800134 )

Reply:
ok, looks unusual. Can you simply prepend every target definition with a line

```
.PHONY: my_target
my_target:
```

?

Reply:
I don't think it will do any effect on wildcard targets, and the rest must be prepended already.